modules/installing-ocp-agent-boot.adoc

```diff
@@ -104,6 +83,9 @@ If you set the platform to `vSphere` or `baremetal`, you can configure IP addres
 * IPv6
 * IPv4 and IPv6 in parallel (dual-stack)
 
+IPv6 is supported only on bare metal platforms.
+====
++
```

Comment:
@skopacz1 I don't know why the following example is even here. Maybe my own ignorance. What is the user supposed to do with it? Is it an optional step they need to perform if using IPv6?

Reply:
I'm also ignorant of why the example exists both here and in the other module, as I didn't write this procedure, but all I know is that Mike Pytlak wanted me to move the examples out of the notes for better page formatting.

If I had to hazard a guess, it's because this entire procedure is basically an example of a basic deployment that users will need to personally modify to meet their own deployment needs, and one of those needs might be using a IPv6 or a combination of v4 and v6. And I'm guessing this example is helpful to the user without having to find some other resource for dual stack networking, like the page linked in the additional resources.

Is it okay if I keep these examples as is and further scrutinize their purpose with SMEs in a later enhancement PR?